server/lib/amplitude.js

```diff
@@ -0,0 +1,245 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This module contains mappings from content server event names to
+// amplitude event definitions. The intention is for the returned
+// `receiveEvent` function to be invoked for every event and the
+// mappings determine which of those will be transformed into an
+// amplitude event.
+//
+// You can read more about the amplitude event
+// structure here:
+//
+// https://amplitude.zendesk.com/hc/en-us/articles/204771828-HTTP-API
+//
+// You can see the event taxonomy here:
+//
+// https://docs.google.com/spreadsheets/d/1G_8OJGOxeWXdGJ1Ugmykk33Zsl-qAQL05CONSeD4Uz4
+
+/* eslint-disable sorting/sort-object-props, camelcase */
+
+'use strict';
+
+const APP_VERSION = /^[0-9]+\.([0-9]+)\./.exec(require('../../package.json').version)[1];
```

Comment:
IIUC, you only want to capture the minor version number?

Reply:
Yep, the train number essentially.